migrate/v0_17/migrate.go

```diff
@@ -0,0 +1,59 @@
+package v0_17
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/kava-labs/kava/app"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
+
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+var (
+	// Configure go live time for the next version of the chain
+	GenesisTime = time.Date(2022, 5, 11, 16, 0, 0, 0, time.UTC)
+	// Configure the id of the next version of the chain
+	ChainID = "kava-10"
```

Comment:
Ditto - the real chain id won't be `kava-10`, lets put a TODO here to change it for the evm chain id when we have it